test/python/transpiler/test_passmanager_run.py

```diff
@@ -83,12 +83,63 @@ def test_default_pass_manager_single(self):
         circuit.cx(qr[1], qr[2])
         circuit.cx(qr[2], qr[3])
 
-        coupling_map = Fake20QV1().configuration().coupling_map
+        coupling_map = [
```

Comment:
can be left for a follow-up but I think these tests can run with a smaller number of qubits (the 20 was kind of arbitrary).

Reply:
The test seems to try to be a challenge for layout and routing, so I prefer to keep it in a bigger backend. However, I used `ALMADEN_CMAP` in 982e274d80572cef4c207eafb6dda7a49250c1b6